### PR TITLE
Fixing error handling for the websocket connection

### PR DESCRIPTION
**Pull request by benthomasson**

### CODE REVIEW

**Comment by benthomasson** [comment]
@cutwater @mkanoor  please review

File: ansible_events/websocket.py [hunk 1]
```diff
@@ -1,22 +1,28 @@
 import json
 import logging
+from asyncio.exceptions import CancelledError
 
 import websockets
 
 
 async def send_event_log_to_websocket(event_log, websocket_address):
-    logger = logging.getLogger()
+    logger = logging.getLogger("ansible_events.websocket")
```
Comment (cutwater) [inline ansible_events/websocket.py:9]: Logger can be initialized globally. No need to perform a lookup at every function call.

File: ansible_events/websocket.py [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
+    logger.info(f"websocket {websocket_address} connecting")
```
Comment (cutwater) [inline ansible_events/websocket.py:10]: Better to pass log message parameters separately and allow logging to format message for you:

```suggestion
    logger.info(f"websocket %s connecting", websocket_address)
```

**Review by cutwater — CHANGES_REQUESTED** [review]

File: ansible_events/websocket.py [hunk 3]
```diff
@@ -1,22 +1,29 @@
...
             while True:
                 event = await event_log.get()
                 await websocket.send(json.dumps(event))
                 if event == dict(type="Shutdown"):
-                    break
+                    return
         except websockets.ConnectionClosed:
-            logger.warning(f"websocket {websocket_address} connection closed")
+            logger.warning("websocket %s connection closed", websocket_address)
             continue
```
Comment (cutwater) [inline ansible_events/websocket.py:23]: Since there is no logic below the `try: ... except:` block, the `continue` statements are redundant here.

**Review by cutwater — CHANGES_REQUESTED** [review]

File: ansible_events/websocket.py [hunk 4]
```diff
@@ -1,22 +1,29 @@
...
                 event = await event_log.get()
                 await websocket.send(json.dumps(event))
                 if event == dict(type="Shutdown"):
-                    break
+                    return
         except websockets.ConnectionClosed:
-            logger.warning(f"websocket {websocket_address} connection closed")
-            continue
+            logger.warning("websocket %s connection closed", websocket_address)
+            pass
```
Comment (cutwater) [inline ansible_events/websocket.py:23]: `pass` statement is not needed as well. The `pass` statement is only useful for empty blocks.

**Review by cutwater — CHANGES_REQUESTED** [review]

**Review by cutwater — APPROVED** [review]

**Pull request merged**